{"errorMessage": "module 'marshmallow' has no attribute '__version_info__'", "errorType": "AttributeError", "requestId": "", "stackTrace": ["  File \"/var/lang/lib/python3.9/importlib/__init__.py\", line 127, in import_module\n    return _bootstrap._gcd_import(name[level:], package, level)\n", "  File \"<frozen importlib._bootstrap>\", line 1030, in _gcd_import\n", "  File \"<frozen importlib._bootstrap>\", line 1007, in _find_and_load\n", "  File \"<frozen importlib._bootstrap>\", line 986, in _find_and_load_unlocked\n", "  File \"<frozen importlib._bootstrap>\", line 680, in _load_unlocked\n", "  File \"<frozen importlib._bootstrap_external>\", line 850, in exec_module\n", "  File \"<frozen importlib._bootstrap>\", line 228, in _call_with_frames_removed\n", "  File \"/var/task/query_handler.py\", line 20, in <module>\n    import pymilvus\n", "  File \"/var/lang/lib/python3.9/site-packages/pymilvus/__init__.py\", line 14, in <module>\n    from .client.prepare import Prepare\n", "  File \"/var/lang/lib/python3.9/site-packages/pymilvus/client/prepare.py\", line 14, in <module>\n    from . import blob, entity_helper, ts_utils\n", "  File \"/var/lang/lib/python3.9/site-packages/pymilvus/client/entity_helper.py\", line 8, in <module>\n    from pymilvus.settings import Config\n", "  File \"/var/lang/lib/python3.9/site-packages/pymilvus/settings.py\", line 4, in <module>\n    import environs\n", "  File \"/var/lang/lib/python3.9/site-packages/environs/__init__.py\", line 58, in <module>\n    _SUPPORTS_LOAD_DEFAULT = ma.__version_info__ >= (3, 13)\n"]}